comps/retrievers/src/README_arango.md

```diff
@@ -0,0 +1,113 @@
+# Retriever Microservice with ArangoDB (work-in-progress)
+
+## 🚀Start Microservice with Python
+
+### Install Requirements
+
+```bash
+pip install -r requirements.txt
+```
+
+### Start ArangoDB Server
+
+To launch ArangoDB locally, first ensure you have docker installed. Then, you can launch the database with the following docker command.
+
+**TODO: Switch to official image when ready**
+
+```bash
+docker run \
+  -p 8529:8529
+  -e ARANGO_ROOT_PASSWORD=test
+  jbajic/arangodb-arm:vector-index-preview
+```
+
+### Setup Environment Variables
+
+```bash
+export no_proxy=${your_no_proxy}
+export http_proxy=${your_http_proxy}
+export https_proxy=${your_http_proxy}
+export ARANGODB_URI=${your_arangodb_uri}
+export ARANGODB_USERNAME=${your_arangodb_username}
+export ARANGODB_PASSWORD=${your_arangodb_password}
+export ARANGODB_DATABASE=${your_arangodb_database}
```

Comment:
Update to use new environment variables (i.e `ARANGO_URL`, `ARANGO_USERNAME`, etc.)